### DISCUSSION TITLE
Environment Set Up, src directory not found

### DISCUSSION THREAD

**Discussion by ahmeedaro** [microsoft/30daysof#127] [Q&A]
Hello, please i cannot find the src directory while trying to follow the environment set-up guide, help, please. See attach image.

![1](https://user-images.githubusercontent.com/24251803/197196181-0aa72c64-522f-4ddf-bee4-329f38a9f181.JPG)

**Comment by BethanyJep** [comment]
Hi, you can follow: [YouTube](https://www.youtube.com/watch?v=6pMvovj7KbE&feature=youtu.be), share if you will still be having the same challenge.
